src/plugins/share/server/url_service/short_urls/short_url_client.ts

```diff
@@ -60,14 +59,13 @@ export class ServerShortUrlClient implements IShortUrlClient {
     locator,
     params,
     slug = '',
-    humanReadableSlug = false,
   }: ShortUrlCreateParams<P>): Promise<ShortUrl<P>> {
     if (slug) {
       validateSlug(slug);
     }
 
     if (!slug) {
-      slug = humanReadableSlug ? generateSlug() : randomStr(4);
+      slug = randomStr(5);
```

Comment:
Why 4 -> 5 :)?

Reply:
Just more entropy so that IDs are less likely to clash.

<img width="433" alt="image" src="https://user-images.githubusercontent.com/82822460/169513974-ec149d9f-8947-43ff-987c-9bba5ac2d801.png">